code/modules/reagents/reagent_containers/dropper.dm

```diff
@@ -31,6 +31,7 @@
 
 				var/time = 20 //2/3rds the time of a syringe
 				user.visible_message("<span class='warning'>[user] is trying to squirt something into [target]'s eyes!</span>")
```

Comment:
Bad copy paste.

Reply:
Yeah; Good Catch. I write the new code under it, and then I cut the previous line, forgot to do it here.